Resources/config/monolog.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\DependencyInjection\Loader\Configurator;
+
+use Monolog\Formatter\ChromePHPFormatter;
+use Monolog\Formatter\GelfMessageFormatter;
+use Monolog\Formatter\HtmlFormatter;
+use Monolog\Formatter\JsonFormatter;
+use Monolog\Formatter\LineFormatter;
+use Monolog\Formatter\LogglyFormatter;
+use Monolog\Formatter\LogstashFormatter;
+use Monolog\Formatter\NormalizerFormatter;
+use Monolog\Formatter\ScalarFormatter;
+use Monolog\Formatter\WildfireFormatter;
+use Monolog\Handler\FingersCrossed\ErrorLevelActivationStrategy;
+use Monolog\Logger;
+use Psr\Log\LoggerInterface;
+use Symfony\Bridge\Monolog\Handler\FingersCrossed\NotFoundActivationStrategy;
+use Symfony\Component\HttpClient\HttpClient;
+use Symfony\Contracts\HttpClient\HttpClientInterface;
+
+return static function (ContainerConfigurator $container) {
+    $container->services()
+
+        ->alias('logger', 'monolog.logger')
+        ->alias(LoggerInterface::class, 'logger')
```

Comment:
`logger` was a public and `LoggerInterface::class` not public. Do you need to make one public or the other private?

Reply:
Sure that `logger` was public? There was no explicit visibility configured and the default should be "not public".